overlays/openstack.yaml

```diff
@@ -0,0 +1,17 @@
+applications:
+  openstack-integrator:
+    charm: openstack-integrator
+    num_units: 1
+    trust: true
+    base: ubuntu@22.04
+  openstack-cloud-controller:
+    charm: openstack-cloud-controller
+    base: ubuntu@22.04
```

Comment:
so, this is a subordinate charm.  I don't think you can set the base b/c the series will be determined by the primary unit (the openstack-integrator)